avm/utilities/e2e-template-assets/templates/diagnostic.dependencies.bicep

```diff
@@ -42,6 +42,9 @@ resource logAnalyticsWorkspace 'Microsoft.OperationalInsights/workspaces@2021-12
 resource eventHubNamespace 'Microsoft.EventHub/namespaces@2021-11-01' = {
   name: eventHubNamespaceName
   location: location
+  properties: {
+    disableLocalAuth: true
+  }
```

Comment:
Inteded? Please not this will also affect all other modules deploying this dependencies file

Reply:
I know. It ends up creating a s360 item for me. I'll remove it from this PR.